flexget/plugins/cli/series.py

```diff
@@ -34,69 +35,66 @@ def display_summary(options):
     :param options: argparse options from the CLI
     """
     session = Session()
```

Comment:
I think we should switch to using the session inside a context manager. (`with Session() as session:`)